pontoon/api/tests/test_views.py

```diff
@@ -74,6 +74,22 @@ def test_locale(django_assert_num_queries):
         "unreviewed_strings": 5,
         "complete": False,
         "projects": ["terminology"],
+        "localizations": [
+            {
+                "terminology": {
+                    "approved_strings": 0,
+                    "complete": True,
+                    "locale": "kg",
```

Comment:
We already have locale in the parent dict, why repeat it in the `localizations` dict?